src/ex3/utils.js

```diff
@@ -0,0 +1,34 @@
+export function capitalize(string) {
+  const updatedString = string.charAt(0).toUpperCase() + string.slice(1);
+  return updatedString;
+}
+
+export function checkItem(item) {
+  const inputArray = item.split(",");
+
+  if (isNaN(+inputArray[0])) {
+    return false;
+  } else {
+    return true;
```

Comment:
if you're going to return true/false then I think it's better to give a name like isItemValid so that this is more readable whre you call the func